### ISSUE TITLE
Add .gitattributes file at the root of the web-application template to normalize line endings when pushing to Git?

### ISSUE THREAD

**Issue by patricklafrance** [workleap/wl-foundry-cli#54]
I am not sure if it' really needed... When I was developing CRACO, I had issues with line endings when I was accepting PR form Unix based OS contributors.

I added a `.gitattributes` file at the root of the repo with the following content to normalize the line endings when pushing to Git:

```
###############################################################################
# Set default behavior to automatically normalize line endings.
###############################################################################
* text=auto eol=lf
```

Should it be added to the CLI templates?

**Comment by patricklafrance** [comment]
@ofrogon @alexasselin008 any comment?

**Comment by ofrogon** [comment]
Fixed in PR #60
